.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
+  "trailingComma": "es5"
+}
```

Comment:
is this needed?

Reply:
We have it in foreman and foreman tasks as well, it helps with formatting

Reply:
I agree, I just wonder if we can have this in our foreman-js linter meta package

Reply:
https://github.com/theforeman/foreman-js/blob/914bdbcba2078436efeef8805ca68011ff2c4c05/packages/eslint-plugin-foreman/lib/src/core.js#L20-L24
Looks like it's there but isn't enforced by prettier